index.js

```diff
@@ -63,11 +63,11 @@ module.exports = function ExpressOpenApi (_routePrefix, _doc, _opts) {
   }
 
   // Validate path middleware
-  middleware.validPath = function (schema = {}) {
+  middleware.validPath = function (schema = {}, pathOpts = {}) {
     let validate
     function validSchemaMiddleware (req, res, next) {
       if (!validate) {
-        validate = makeValidator(middleware, getSchema(validSchemaMiddleware), opts)
+        validate = makeValidator(middleware, getSchema(validSchemaMiddleware), { ...pathOpts, ...opts })
```

Comment:
If this is just merging them back together could we just drop the new additional argument?

Reply:
Oh wait, `opts` here is from somewhere else. Ah, I see the reason now. Sorry, first week back from a month of vacation so my brain is not fully back in a detail oriented mode.